### ISSUE TITLE
sign up steps failed

### ISSUE THREAD

**Issue by johnfelipe** [sqlpad/sqlpad#1131]
`docker run -p 3000:3000 --volume ~/docker-volumes/sqlpad-oracle:/var/lib/sqlpad --detach sqlpad/sqlpad:latest`

and next show me forbiden

![SNAG-1255](https://user-images.githubusercontent.com/428820/173157310-73668e30-48fc-40e8-9daa-588cc25b5d95.png)

**Comment by rickbergfalk** [comment]
Hi @johnfelipe. Sign up functionality is only allowed for email addresses that have been "invited" to SQLPad. 

For first account, you'll need to provide the email address and password using environment variables: https://getsqlpad.com/#/authentication?id=local-authentication

**Comment by johnfelipe** [comment]
Is in an env file or how set admin user and admin pwd?

**Comment by rickbergfalk** [comment]
Yes there are environment variables SQLPAD_ADMIN and SQLPAD_ADMIN_PASSWORD.

You can set them any way you prefer to set them for docker if using that. Otherwise there is an env file you can provide https://getsqlpad.com/#/configuration

**Comment by johnfelipe** [comment]
pls help me, i cant login
